src/common/__tests__/data/locations.ts

```diff
@@ -153,3 +154,16 @@ export const sec555RoomResponse: RoomAdminResponse = {
     },
   },
 };
+
+export const updateSEC555Room: UpdateRoom = {
+  id: secRoomResponse.id,
+  name: '555a',
+  capacity: 95,
+};
+
+export const updateSECRoomResponse: RoomAdminResponse = {
```

Comment:
Needs a quick docblock describing the test cases this is used for

Reply:
And also added in a docblock for this through [1bf3981](https://github.com/seas-computing/course-planner/pull/581/commits/1bf398187801902dbed320b8e7f7e1bb5e890888)..